tidb-cloud/migrate-from-oracle-using-aws-dms.md

```diff
@@ -0,0 +1,152 @@
+
+5. Click the **Connect** button to connect your TiDB cluster:
+
+    ![Connect to TiDB Cloud Serverless Tier](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-7.png)
+
+## Step 5. Create an AWS DMS replication instance
+
+1. Go to the [Replication instances](https://console.aws.amazon.com/dms/v2/home#replicationInstances) page in the AWS DMS console, and switch to the corresponding region.
+
+2. Create an AWS DMS replication instance with `dms.t3.large` in the VPC.
+
+    ![Create AWS DMS Instance](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-8.png)
+
+## Step 6. Create DMS endpoints
+
+1. In the [AWS DMS console](https://console.aws.amazon.com/dms/v2/home), click the replication instance that you just created.
+
+2. Create the Oracle source endpoint and the TiDB target endpoint.
+
+    The following screenshot shows the configurations of the source endpoint.
+
+    ![Create AWS DMS Source endpoint](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-9.png)
+
+    The following screenshot shows the configurations of the target endpoint.
+
+    ![Create AWS DMS Target endpoint](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-10.png)
+
+## Step 7. Migrate the schema
+
+In this example, AWS DMS automatically handles the schema, since the schema definition is simple.
+
+If you decide to migrate schema using the AWS Schema Conversion Tool, see [Installing AWS SCT](https://docs.aws.amazon.com/SchemaConversionTool/latest/userguide/CHAP_Installing.html#CHAP_Installing.Procedure).
+
+For more information, see [Migrating your source schema to your target database using AWS SCT](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_GettingStarted.SCT.html).
+
+## Step 8. Create a database migration task
+
+1. In the AWS DMS console, go to the [Data migration tasks](https://console.aws.amazon.com/dms/v2/home#tasks) page. Switch to your region. Then click **Create task** in the upper right corner of the window.
+
+    ![Create task](/media/tidb-cloud/aws-dms-to-tidb-cloud-create-task.png)
+
+2. Create a database migration task and specify the **Selection rules**:
+
+    ![Create AWS DMS migration task](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-11.png)
+
+    ![AWS DMS migration task selection rules](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-12.png)
+
+3. Create the task, start it, and then wait for the task to finish.
+
+4. Click the **Table statistics** to check the table. The schema name is `ADMIN`.
+
+    ![Check AWS DMS migration task](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-13.png)
+
+## Step 9. Check data in the downstream TiDB cluster
+
+Connect to the [Serverless Tier cluster](https://tidbcloud.com/console/clusters/create-cluster) and check the `admin.github_event` table data. As shown in the following screenshot, DMS successfully migrated table `github_events` and 10000 rows of data.
+
+![Check Data In TiDB](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-14.png)
```

Comment:
/media/tidb-cloud/aws-dms-from-oracle-to-tidb-14.png image has mysql ERROR 2013 please remove that from the screenshot .

Reply:
nice catch, done.